backend/src/main/java/se/chalmers/dat265/group1/api/customer/CustomerAPI.java

```diff
@@ -1,6 +1,7 @@
 package se.chalmers.dat265.group1.api.customer;
 
 import se.chalmers.dat265.group1.model.*;
+import se.chalmers.dat265.group1.model.dbEntities.OrderedPart;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'se.chalmers.dat265.group1.model.dbEntities.OrderedPart'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)